app/src/main/java/com/kamron/pogoiv/activities/OcrCalibrationResultActivity.java

```diff
@@ -105,7 +105,11 @@ public static void startCalibration(@NonNull Context context,
             sCalibrationImage = bitmap;
         } else {
             // Make a mutable copy of the bitmap so we can draw on it with a Canvas
-            sCalibrationImage = bitmap.copy(sCalibrationImage.getConfig() ,true);
+            sCalibrationImage = bitmap.copy(Bitmap.Config.ARGB_8888 ,true);
```

Comment:
Using `Bitmap.Config.ARGB_8888` might be a waste of space if the screenshot bitmap used a lower color configuration.
See the previous suggestion to copy the configuration from the screenshot bitmap.